From 80ea699313b19c930218daa2f11f1d48c2edbd06 Mon Sep 17 00:00:00 2001
From: Joey Riches <josephriches@gmail.com>
Date: Wed, 1 Nov 2023 11:59:36 +0000
Subject: [PATCH 1/1] Fix appstream generation with appstream-builder

- appstream-builder can't handle remote icons, needs the local icon.
---
 data/ibus-anthy.appdata.xml.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/data/ibus-anthy.appdata.xml.in b/data/ibus-anthy.appdata.xml.in
index af32a5a..22822aa 100644
--- a/data/ibus-anthy.appdata.xml.in
+++ b/data/ibus-anthy.appdata.xml.in
@@ -6,7 +6,6 @@
   <project_license>GPL-2.0</project_license>
   <name translatable="no">Anthy</name>
   <summary>Japanese input method</summary>
-  <icon type="remote" width="48" height="48">https://raw.githubusercontent.com/ibus/ibus-anthy/main/icons/ibus-anthy.png</icon>
   <description>
     <p>
       The Anthy input method is designed for entering Japanese text.
-- 
2.42.0

